tests/scripts/fci.sh

```diff
@@ -3,22 +3,16 @@
 # # # This source code is subject to the license referenced at
 # # # https://github.com/NRLMMD-GEOIPS.
 
-# geoips run single_source $GEOIPS_TESTDATA_DIR/fci_test/20250211.0010/*.nc \
 geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_geocolor/data/mtg/20240924.1500/*.nc \
     --reader_name fci_netcdf \
     --reader_kwargs '{"self_register": "LOW"}' \
     --self_register_dataset 'FULL_DISK' \
     --self_register_source fci \
     --output_formatter unprojected_image \
-    --output_formatter_kwargs '{"x_size": "11136", "y_size": "11136"}' \
+    --output_formatter_kwargs '{"x_size": "5568", "y_size": "5568"}' \
```

Comment:
Could probably make this even smaller if we are changing it ?

Reply:
500x500?